spec/system/dashboard_system_spec.rb

```diff
@@ -549,7 +546,7 @@
 
             it "has a widget displaying the Diaper drive totals from yesterday, only using donations from yesterday" do
               within "#diaper_drives" do
-                expect(page).to have_content(/2 diaper drive/i)
+                expect(page).to have_content(/1 diaper drive participant/i)
```

Comment:
Why did this expectation end up changing?

Reply:
Based on our reading of the test fixtures and the description, we believe that it was intended to only have 1 participant. But we are open to correction if we misread!

Reply:
I provided more context in [the specific commit message](https://github.com/rubyforgood/diaper/pull/1369/commits/1487d75dfbf3a4154ec1e36a9d933c9f9e46c75e), but essentially this context goes through the trouble of [destroying all prior records](https://github.com/rubyforgood/diaper/pull/1369/files#diff-16eb91a445181dca6aa1618ac37f2d40R479) and then specifically only creates [one record for yesterday](https://github.com/rubyforgood/diaper/pull/1369/files#diff-16eb91a445181dca6aa1618ac37f2d40R486). Based on my reading of the spec, there should only be one record, but I'm happy to correct it if I missed something.

Reply:
Ah, I bet it's that, awesome catch :D